client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -43,13 +43,37 @@ function createQuestionEpic(action$, state$, { window }) {
     ofType(actionTypes.createQuestion),
     tap(() => {
       const state = state$.value;
-      const challengeFiles = challengeFilesSelector(state);
+      let challengeFiles = challengeFilesSelector(state);
       const {
         title: challengeTitle,
         superBlock,
         block,
         helpCategory
       } = challengeMetaSelector(state);
+      if (
+        challengeFiles?.some(file => file.editableRegionBoundaries.length > 0)
+      ) {
+        const editableRegionStrings = fileExtension => {
+          const startComment = fileExtension === 'html' ? '<!--' : '/*';
+          const endComment = fileExtension === 'html' ? '-->' : '*/';
+          return `\n${startComment} 'User Editable Region' ${endComment}\n`;
+        };
+
+        const filesWithEditableRegions = challengeFiles.map(file => {
+          const { contents: Contents, editableRegionBoundaries, ext } = file;
+          if (editableRegionBoundaries.length > 0) {
+            const comment = editableRegionStrings(ext);
+            const [start, end] = editableRegionBoundaries;
+            const editableContents = Contents.split('\n');
+            editableContents.splice(start, 0, comment);
+            editableContents.splice(end, 0, comment);
+            const contents = editableContents.join('\n');
```

Comment:
```suggestion
            const lines = contents.split('\n');
            lines.splice(start, 0, comment);
            lines.splice(end, 0, comment);
```
It's a little misleading, since `editableContents` has a specific meaning in the codebase - literally the part of the contents that is inside the editable region.